.github/workflows/tests.yml

```diff
@@ -139,6 +139,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: 1.61.0
+            override: true
```

Comment:
After toolchain installation, I recommend adding <code>- uses: [Swatinem/rust-cache@v2](https://github.com/Swatinem/rust-cache#readme)</code> as a step, which will take care of caching dependencies.